<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmTypeWidget</class>
 <widget class="QWidget" name="AlarmTypeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>159</width>
    <height>132</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="whatsThis">
      <string>Select which alarm type(s) this resource should contain.</string>
     </property>
     <property name="title">
      <string>Alarm Types</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QCheckBox" name="activeCheckBox">
        <property name="text">
         <string>Active Alarms</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="archivedCheckBox">
        <property name="text">
         <string>Archived Alarms</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="templateCheckBox">
        <property name="text">
         <string>Alarm Templates</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
